qcore/formats.py

```diff
@@ -0,0 +1,29 @@
+"""
+Functions and classes to load data that doesn't belong elsewhere.
+"""
+
+import numpy as np
+
+def load_im_file(csv_file, all_psa = False):
+
+    # process column names
+    use_cols = []
+    col_names = []
+    with open(csv_file, 'r') as f:
+        raw_cols = f.readline().strip().split(',')
```

Comment:
what do you think of using this instead?
f.readline().replace(" ","").split(',')